budget/app.py

```diff
@@ -1,14 +1,176 @@
 import json
 import logging
 
+import boto3
+from botocore.exceptions import ClientError
+import requests
```

Comment:
import synapseclient

Reply:
Was deliberately avoiding using that because I don't have to auth.

Reply:
https://sagebionetworks.jira.com/browse/SYNPY-1084

Reply:
Great, thanks!